calendar.rb

```diff
@@ -0,0 +1,203 @@
+require 'date'
+require_relative 'event_info'
+require 'colorize'
+
+class Calendar
+  def initialize
+    @all_events = {}
+  end
+
+  def put_event date
+    count = 0
+    @all_events.each do |key, value|
+      if key == date
+        count = value.length
+      end
+    end
+    count
+  end
+
+  def show_calendar(month, year)
+    total_days = Date.new(year,month,-1).mday
+    tabs = Date.parse(Time.now.strftime("01/%m/%Y")).wday
+    puts "S\tM\tT\tW\tT\tF\tS\t".blue
+    @i = 0
+
+    tabs.times do 
+      print "\t"
+      @i += 1
+    end
+
+    total_days.times do |day|
+      count = put_event("#{day + 1}-#{month}-#{year}")
+      if count > 0
+        print "#{day + 1}(#{count})\t".blue
+      else
+        print "#{day + 1} \t"
+      end
+      @i += 1
+      if @i % 7 == 0
```

Comment:
you don't need this counter. check from the date object if it's Saturday then enter a line break.